NAMESPACE

```diff
@@ -20,6 +20,7 @@ S3method(label,yspec)
 S3method(long,ycol)
 S3method(long,yspec)
 S3method(lookup,ycol)
+S3method(merge,list)
```

Comment:
I'm guessing this was done to avoid this warning:

```
$ Rscript -e 'devtools::document()'
ℹ Updating yspec documentation
ℹ Loading yspec
✖ utils.R:3: S3 method `merge.list` needs @export or @exportS3method tag.
```

I don't think this previously internal helper should be exported as an S3 method for `merge`.

What about renaming `merge.list` to `merge_list` instead (and dropping the `@export`)?

Reply:
I'm going to do this in another PR if possible.